Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsController.cs

```diff
@@ -32,7 +32,7 @@ public AnalyticsController(
         public void Initialize(IWeb3Identity? web3Identity)
         {
             TrackSystemInfo();
-            analytics.Identify(web3Identity?.Address ?? "not cached");
+            analytics.Identify(web3Identity?.Address ?? "not cached", null); //TODO provide logic
```

Comment:
We should not send the identify here, the identify must be used only when the user is authenticated. What you can do here instead is:

1. Initialize anonymousId for the user.
2. Send all tracking events with the anonymousID as payload.
3. When user is identified, send the identify ID with the user's address.
4. All tracking events should add the user_id with the wallet id.